entrypoint.sh

```diff
@@ -2,12 +2,12 @@
 
 printenv >> /etc/environment
 ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-grpcurl -plaintext -protoset-out dish.protoset 192.168.100.1:9200 describe SpaceX.API.Device.Device
+grpcurl -plaintext -protoset-out dish.protoset 192.168.100.1:9200 describe SpaceX.API.Device.Device > /dev/null
```

Comment:
The grpcurl command is only needed to generate the dish.protoset file, no need to see its output each time the container is started.